tests/rkt_net_test.go

```diff
@@ -608,6 +608,10 @@ func testNetCustomNatConnectivity(t *testing.T, nt networkTemplateT) {
 
 	// Child connects to host
 	hostname, err := os.Hostname()
+	if err != nil {
+		t.Fatalf("os.Hostname() got error %q", err)
```

Comment:
If you mean deleting temporary files and rkt context, defer is still executed after a `panic`, but since we don't use any stack trace prettifiers for panics, I thought this way would be much more readable.